tests/__init__.py

```diff
@@ -21,6 +22,21 @@
 WINDOWS = os.name == "nt" or "cygwin" in platform.system().lower()
 PD_LTE_15 = Version(pd.__version__) < Version("1.5.999")
 
+arrow_skip = pytest.mark.skipif(
+    sys.version_info >= (3, 11), reason="pyarrow is not available for 3.11 yet"
+)
+"""This is only needed temporarily due to no wheels being available for arrow on 3.11"""
```

Comment:
Should this be a comment rather than a song?

Reply:
I think you meant "string", not "song", but fixed in next commit

Reply:
Yes,